Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -100,12 +111,31 @@ final class PledgeCTAContainerView: UIView {
 
 // MARK: - Styles
 
-private let projectStateButtonStyle: ButtonStyle = { (button: UIButton) in
-  button
-    |> roundedStyle(cornerRadius: 12)
-    |> UIButton.lens.titleColor(for: .normal) .~ .white
-    |> UIButton.lens.titleLabel.font .~ .ksr_headline(size: 15)
-    |> UIButton.lens.layer.borderWidth .~ 0
-    |> UIButton.lens.titleEdgeInsets .~ .init(topBottom: Styles.grid(1), leftRight: Styles.grid(2))
-    |> (UIButton.lens.titleLabel .. UILabel.lens.lineBreakMode) .~ .byWordWrapping
+private func adaptableStackViewStyle(_ isAccessibilityCategory: Bool) -> (StackViewStyle) {
```

Comment:
Using private style because we were only using 50% of the global checkout style which we would use and then had to change 2 props back to default...so this should be simpler